packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -54,10 +58,6 @@
 
 * Updates Guava version to 32.0.1.
 
-## 0.5.0+6
```

Comment:
Definitely do not delete information from the changelog.

Reply:
ok, restored

Reply:
too much merges since start, accidentally deleted it.